neqo-transport/src/tracking.rs

```diff
@@ -593,11 +603,9 @@ impl AckTracker {
             .ack_freq(seqno, tolerance, delay, ignore_order);
     }
 
-    // Force an ACK to be generated immediately (a PING was received).
-    pub fn immediate_ack(&mut self, now: Instant) {
-        self.get_mut(PacketNumberSpace::ApplicationData)
-            .unwrap()
-            .immediate_ack(now);
+    // Force an ACK to be generated immediately.
```

Comment:
I believe the doc comment below also needs updating:

https://github.com/mozilla/neqo/blob/0eb9174d7a67b250607f0c3ea056fe056bcf91f5/neqo-transport/src/tracking.rs#L420-L424